completion_aggregator/management/commands/run_aggregator_service.py

```diff
@@ -30,8 +30,8 @@ def add_arguments(self, parser):
         """
         parser.add_argument(
             '--batch-size',
-            help='Maximum number of CourseModuleCompletions to migrate, per celery task. (default: 10000)',
-            default=10000,
+            help='Maximum number of StaleCompletions to process, per celery task. (default: 1000)',
+            default=1000,
```

Comment:
Documentation cleanup, and modifying a default value that we had to override in practice, because the larger value caused slow queries and database lock contention.